suse_openstack_cloud

```diff
@@ -195,13 +195,13 @@ for thisrpm in $(cat "$rpm_list"); do
     validate_rpm_if_installed "$thisrpm"
 done
 
-_find_uncompressed () {
-    find "$@" ! -name \*.gz ! -name \*.bz2 ! -name \*.xz
+_find_text_files () {
+    find "$@" -exec grep -Iq . {} \; -and -print
```

Comment:
we should add a -type f here or it will try to grep directories.
and the grep seems to return true on some binary files, too (e.g. try running it on a .git dir)